package.json

```
@@ -1,9 +1,12 @@
 {
     "name": "web3",
     "namespace": "ethereum",
-    "version": "1.0.0-beta.55",
+    "version": "2.0.0-alpha",
     "description": "Ethereum JavaScript API wrapper repository",
     "license": "LGPL-3.0",
+    "engines": {
+        "node": "<8.0.0 <12.0.0"
```

```suggestion
        "node": ">=8.0.0 <12.0.0"
```

Is the `<8.0.0` a typo?